.github/workflows/ios-artifact-build.yml

```diff
@@ -58,9 +58,23 @@ jobs:
           git add .
           git commit --allow-empty -m "chore(#891): changes in script"
 
-      - name: Push Changes to Specific Branch
-        run: |
-          git pull origin --rebase
-          git push
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5.0.2
+        with:
+          commit-message: creating swift artifact
+          title: Release changes
+          body: Automated PR for swift artifact build.
+          branch: "update-ios-artifact"
+          delete-branch: true
+          base:${{ github.head_ref }}
+          token: ${{ secrets.ACTION_PAT }}
+          signoff: true
+
+      - uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          fields: repo,message,author,commit,workflow,job # selectable (default: repo,message)
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
Use Inji mobile team slack channel. secrets for that is SLACK_WEBHOOK_INJI_TEAM

Reply:
done